admin-manual/maintenance/cli-import-export.rst

```diff
@@ -1804,7 +1813,7 @@ log should be output. Example:
 
 .. code-block:: bash
 
-   php symfony physicalobject:import --error-log="/usr/share/nginx/atom/import-log.txt" lib/task/import/example/example_physicalobject.csv
+   php symfony physicalobject:import --error-log="/usr/share/nginx/atom/import/log.txt" lib/task/import/example/example_physicalobject.csv
```

Comment:
This looks like a mistake, perhaps? changing the dash to a slash.... there is no `import` directory below the root AtoM directory, so this would result in an error, i suspect

Reply:
Also a victim of my code-block "cleanup". Thanks for your eagle-eye. Fixed it.